sherpa/models/tests/test_model_op.py

```diff
@@ -110,18 +105,18 @@ def test_basic_binop_raw(op):
     assert isinstance(mdl, BinaryOpModel)
     assert mdl.name == 'polynom2d xOx gauss2d'
     assert mdl.op == op
-    assert mdl.opstr == 'xOx'
     assert len(mdl.parts) == 2
     assert mdl.parts[0] == l
     assert mdl.parts[1] == r
     assert mdl.ndim == 2
 
 
 @pytest.mark.parametrize("op,opstr",
-                         [(np.add, '+'), (np.multiply, '*'),
-                          (np.subtract, '-'), (np.divide, '/'),
-                          (np.floor_divide, '//'), (np.true_divide, '/'),
-                          (np.remainder, '%'), (np.power, '**')])
+                         [(operator.add, '+'), (operator.mul, '*'),
+                          (operator.sub, '-'),
+                          (operator.floordiv, '//'), (operator.truediv, '/'),
+                          (operator.mod, '%'), (operator.pow, '**')],
+                          )
```

Comment:
Should we still test the `np.XXX` versions (e.g. add in the `operator.XXX` versions to the tests rather than replace the NumPy ones)?